src/Microsoft.Performance.SDK.Tests/TableDescriptorUtils.cs

```diff
@@ -15,7 +15,7 @@ internal static class TableDescriptorUtils
             ITableConfigurationsSerializer serializer,
             params Type[] types)
         {
-            CreateTableDescriptors(serializer, out var descriptors, out _, out _, types);
+            CreateTableDescriptors(serializer, out var descriptors, out var _, types);
```

Comment:
nit: out _ is the syntax for discarding an out parameter (if we are truly not using it)

https://docs.microsoft.com/en-US/dotnet/csharp/fundamentals/functional/discards

Reply:
thx